modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase

```diff
@@ -1,28 +1,166 @@
 @component-name = "portal-commerce"
 definition {
 
+	property custom.properties = "jsonws.web.service.paths.excludes=";
 	property portal.release = "true";
 	property portal.upstream = "true";
 	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
 	property testray.main.component.name = "Product Info Management";
 
 	setUp {
-		CommerceConfiguration.commerceSetUp(storeFront = "true");
+		CommerceConfiguration.commerceSetUp(
+			minium = "true",
+			storeFront = "true");
 	}
 
 	tearDown {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-10491. The product can be added to cart until there's stock quantity"
+	@priority = "3"
+	test CanOnlyPlaceOrderWhenStockIsAvailable {
+		property portal.acceptance = "true";
```

Comment:
Are these tests supposed to be in the relevant? Just asking, as it has "true".

Reply:
No they don't, property was remove.

Except the "UseConfiguration" this test case needs to be in the relevant.